packages/generator-langium/langium-template/package.json

```diff
@@ -29,7 +29,7 @@
     "langium": {
         "languageId": "<%= language-id %>",
         "grammar": "src/language-server/<%= language-id %>.langium",
-        "extensions": <%= file-extension %>,
+        "fileExtensions": <%= file-extension %>,
```

Comment:
This needs an entry in the `CHANGELOG.md`. Could you create one in `langium-cli`?

You can take https://github.com/eclipse/lsp4j/blob/master/CHANGELOG.md as an example format.